server/routes/CoursePage.js

```diff
@@ -0,0 +1,44 @@
+// Assuming `studentRouter` is already defined
+import Course from "../models/Course.js"; // Import the Course model
+
+studentRouter.get("/courses", async (req, res) => {
+  const { user_id } = req.query;
+
+  if (!user_id) {
+    return res
+      .status(400)
+      .json({ message: "User ID is required to fetch courses" });
+  }
+
+  try {
+    // Fetch courses for the student by user_id, assuming user_id is a field in your student data
+    const courses = await Course.find(
+      { student_id: user_id },
+      "course_id number"
+    ); // Adjust field names as per your schema
```

Comment:
use the functions defined in db object, if you cannot find a function suited for your needs, create one!
Because at this point, you don't have an assured connection to the db